reference-service/src/main/java/com/transformuk/hee/tis/reference/service/service/mapper/FundingTypeMapper.java

```diff
@@ -18,4 +22,10 @@ public interface FundingTypeMapper {
   FundingType fundingTypeDTOToFundingType(FundingTypeDTO fundingTypeDTO);
 
   List<FundingType> fundingTypeDTOsToFundingTypes(List<FundingTypeDTO> fundingTypeDTOs);
+
+  @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
+  @Mapping(target = "id", ignore = true)
+  @Mapping(target = "uuid", ignore = true)
+  @Mapping(target = "fundingSubTypes", ignore = true)
```

Comment:
It's good to think about the integrity of these fields.  As well as/rather than 'ignoring' identifiers, we should consider verifying the consistency with the entity being updated. i.e. we already verify consistency of the `id` by using it for a search.  A `uuid` which doesn't match may indicate an issue with the object sent by a client or it may be the purpose of the update.

It seems odd/confusing to have annotations for `fundingSubTypes` when that doesn't exist on the source class.